src/ui/public/filters/_prop_filter.js

```diff
@@ -38,14 +38,21 @@ export function propFilter(prop) {
    * @return {array} - the filtered list
    */
   return function (list, filters) {
-    if (!filters) return filters;
+    if (!filters) {
```

Comment:
Since we are anyway cleaning this up, I would suggest, that this function should also work with an empty array. I would suggest we use `filters = []` above and check for the array length here (and add a test for that).